imgparse/imgparse.py

```diff
@@ -87,11 +87,15 @@ def get_ils(image_path=None, xmp_data=None):
 
     :param image_path: the full path to the image (optional if `xmp_data` provided)
     :param xmp_data: the XMP data of image, as a string dump of the original XML (optional to speed up processing)
+    :param use_clear_channel: if true, refer to the ILS clear channel value instead of the default
```

Comment:
I just have one question about this -- if the sensor that captured the image is a D4K, is it even possible for it to have an ILS value as defined for the 6X (i.e. independent values for each channel)? Or will it _always_ be better to use the ILS:Clear value?

Reply:
I believe that it's _always_ better to use the clear value. @ZachMarston may be able to answer more definitively.

Reply:
Got it. What would you think about having this function determine the make/model of the camera (by calling `get_make_and_model` internally), and choosing the correct ILS value based on this? It would really only make sense if 6X imagers _always_ want to use the standard `SunSensor` values, and D4K imagers _always_ want to use the ILS:Clear values. If there's ever situations where one might want to use the other, I agree that an extra argument is better.

Reply:
Are any other sensors (outside of 6x and d4k) supported by this library?

Reply:
It was written specifically for the 6X (you can see it in the docstring actually), but that's because we thought that was the only sensor that supported an ILS module. It technically should support any sensor that can have an ILS module; currently, I believe that is only the 6X and D4K, but it might not always be that way in the future.

Reply:
Hmm. My instinct is to let the user decide which piece of ils data they want, but I see the appeal of having the function determine it as well. I would rather keep it as is, but I'm only 60-40 on it. Do you feel strongly about doing it the other way?

Reply:
Nope, as-is sounds fine 👍